tensorflow/lite/kernels/unpack_test.cc

```diff
@@ -67,192 +66,97 @@ class UnpackOpModel : public SingleOpModel {
   std::vector<int> outputs_;
 };
 
-// float32 tests.
-TEST(UnpackOpTest, FloatThreeOutputs) {
-  UnpackOpModel<float> model({TensorType_FLOAT32, {3, 2}}, 0);
-  model.SetInput({1, 2, 3, 4, 5, 6});
-  model.Invoke();
+template <typename T>
+void Check(const int axis, const std::initializer_list<int> input_shape,
```

Comment:
does const reference not work?

Reply:
const reference fix the changes, sorry for delay.